portality/templates/includes/header.html

```diff
@@ -41,12 +40,12 @@ <h2 class="sr-only">Main actions</h2>
         {% set extraClass="secondary-nav--precedes-highlight" %}
     {% endif %}
 {% endif %}
-<nav class="secondary-nav {{ extraClass }}">
+<nav class="secondary-nav {{ extraClass }}" aria-label="Secondary Nav">
```

Comment:
Same here; the heading associated for the list indicates that it’s the Menu, so we should keep them that way (Secondary nav isn’t a helpful label for the user)

Reply:
Yes, that should state "Secondary Menu".

Reply:
It’s more for consistency here; the corresponding heading (`<h2 class="secondary-nav__header">Menu</h2>`) says _Menu_ only -- I chose this as this was the main menu content (and _Main actions_ seemed more semantically correct for the top nav items, "Support" and "Apply" being action verbs). The shorter the label, the better.